packages/create-cloudflare/templates/angular/c3.ts

```diff
@@ -22,7 +21,7 @@ const configure = async (ctx: C3Context) => {
 	await installCFWorker(ctx);
 };
 
-async function installCFWorker(ctx: C3Context) {
+async function installCFWorker(_ctx: C3Context) {
```

Comment:
Why not just delete this ad remove it from the call above?

Reply:
my bad! thanks 😄